### ISSUE TITLE
autoclick doesn't disable on x00 bosses?

### ISSUE THREAD

**Issue by Charoncaori** [wchill/steamSummerMinigame#268]
Hey, I updated to your latest version, and this bug is still here from earlier today. Whenever I get to the boss, my screen still shows click damage happening (I don't have any autofire cannons bought, and I get money from raining gold, so it has to be click damage). Someone said it was just client side, but my money is still going up, so I think it's a proper bug. 
(http://i.imgur.com/NO8750g.png)

**Comment by ryantheleach** [comment]
Are you running any other scripts? Others are reporting this happens if you have 0 wormholes.

**Comment by ensingm2** [comment]
No, it's us. We don't disable clicking currently. I'm planning on adding a 'WormHoleStrats' setting that is toggleable and will do this though (Kind of swamped with other stuff at the moment however). 

I just didn't want to add it in with no toggle as that effectively screws anyone that isn't in a wormhole room. As long as you're not nuking, which you aren't, clicking isn't THAT detrimental to WHStrats. WHStrats are just being fucked over by people that nuke/cripple exclusively on 100 levels.

**Comment by lunboks** [comment]
Happening to me too since I ran out of wormholes.

**Comment by g1real** [comment]
Not a bug, intended
https://github.com/wchill/steamSummerMinigame/blob/master/autoPlay.user.js#L833

Doesn't really matter since DPS is irrelevant though.

**Comment by K4r0qtuYNE5G4qgZ** [comment]
About Autoclick, i watched why some Bosses die by it, alsways if goldrain active on boss it goes down faster, i suppose its because of those scripts that only click like hell if there is gold to be earned... Many User uses them because they say (its not cheat if only autoclick for gold) but if suddenly 500 with high Click Damage starts clicking like hell on the Boss ... it goes down faster.

I analized it yesterday on a >100m Room, as soon as there are no goldrain in Abilities, Boss stays, else goes down slowly but still...
